src/resources/line_item.rs

```diff
@@ -91,7 +95,7 @@ impl Object for InvoiceLineItem {
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
-pub struct TaxAmount {
+pub struct InvoiceTaxAmount {
```

Comment:
Could rename this struct and/or break it out into the shared `src/resources/types.rs` to:

- De-duplicate it between `LineItem` and `Invoice`.
- Renaming it either way to avoid the breaking change to the rust API.